README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## __Dominique Janvier__
+
+![_Headshot taken by my wonderful friend Amy Tian_](https://github.com/djanvier/hw01/blob/master/DominiqueHeadshot.jpg)
+
+
+### **About me**
+Salut! My name is Dominique Janvier, and I am a third-year Public Policy major/French minor at the University of Chicago. I am originally from Durham, North Carolina, a city located in the Triangle area (adjascent to Raleigh and Chapel Hill) in the Piedmont region of North Carolina. I am a Haitian-American and a child of immigrants, so a lot of my ubringing was informed by my cultural identity as a Haitian-American and the values that come with it. I have always had an inifity for the arts, thus many of my hobbies growing up consisted of singing, painting, dancing, and acting/theatre, including an 8-year stint as a choir nerd! In addition to music, I played soccer for 12 years (up until my ACL injury during my junior year of high school), and was heavily involved in STEM through summer camps, clubs, and research. Currently at UChicago, I participate in a variety of RSO's across a wide-range of disciplines, including Doc Films, Peer Health Advocates, and Leaders of Color at the Institue of Politics. 
+
+### **Favorites**
```

Comment:
I recommend leaving a line break between headers and the following content. It will prevent rendering issues from occurring once we start using R Markdown documents.

```
### **Favorites**

* Favorite Color: Green
* Favorite Movie: Billy Elliot
* Favorite Food: Spaghetti
* Favorite Music Genre: Indie/Alt Rock
```